package.json

```diff
@@ -24,7 +24,7 @@
     "dist"
   ],
   "scripts": {
-    "build": "tsc --project tsconfig.build.json",
+    "build": "tsc --project tsconfig.build.json && chmod +x dist/bin/ember-codemod-pod-to-octane.js",
```

Comment:
TypeScript doesn't allow changing file permissions. We can use `chmod` to separate concerns.